src/complete.ts

```diff
@@ -52,10 +82,25 @@ const Span = /^\w*$/, QuotedSpan = /^[`'"]?\w*[`'"]?$/
 export function completeFromSchema(schema: {[table: string]: readonly (string | Completion)[]},
                                    tables?: readonly Completion[],
                                    defaultTable?: string): CompletionSource {
+  let bySchema: {[schema: string]: readonly Completion[]} = Object.create(null)
```

Comment:
This is used to create autocomplete for tables after user has typed a schema name, e.g. `public.u|` would return `users` completion item.